### ISSUE TITLE
Unable to resolve "@shopify/react-native-skia" from "node_modules/wrn-echarts/src/skiaChart.tsx"

### ISSUE THREAD

**Issue by pavelbabenko** [wuba/wrn-echarts#13]
Cannot use the library with just react-native-svg installed. It require skia to be installed.
Unable to resolve "@shopify/react-native-skia" from "node_modules/wrn-echarts/src/skiaChart.tsx"

**Comment by zhiqingchen** [comment]
have you try

```js

import SvgChart, { SVGRenderer } from 'wrn-echarts/lib/module/svgChart';

```

**Comment by pavelbabenko** [comment]
@zhiqingchen It works. Unfortunately, typescript shows an error
`Could not find a declaration file for module 'wrn-echarts/lib/module/svgChart'.`

**Comment by zhiqingchen** [comment]
> @zhiqingchen It works, thank you!. Unfortunately, typescript shows an error `Could not find a declaration file for module 'wrn-echarts/lib/module/svgChart'.`


@pavelbabenko , Thanks for the feedback, The type bug will be fixed in a future release.

**Comment by zhiqingchen** [comment]
@pavelbabenko 
The problem is fixed in 0.1.5 and can be import like this
```tsx
import SvgChart, { SVGRenderer } from 'wrn-echarts/svgChart';
```

https://wuba.github.io/wrn-echarts/docs/advanced-guides/import-individually/
